src/model/Ticket.java

```diff
@@ -24,7 +30,7 @@ public Ticket(String concertHall, String eventCode, Instant time) {
         this.time = time;
     }
 
-    public Ticket(String id, String concertHall,
+    public Ticket(UUID id, String concertHall,
```

Comment:
You do not need to use "UUID id" here as argument. this field will fill with generated value from constructor in BaseEntity

Reply:
Yes, sure. I left it in case automatical generated id will have been be a wrong solution for the project.

Reply:
So, I deleted this extra "UUID id"

Reply:
no, it is good solution. approved